docs/concepts/events.md

```diff
@@ -1,148 +1,183 @@
-Events
-======
+# Events
 
-Forge uses an event bus that allows mods to intercept events from various Vanilla and mod behaviors.
+One of NeoForge's main features is the event system. Events are fired for various things that happen in the game. For example, there are events for when the player right clicks, when a player or another entity jumps, when blocks are rendered, when the game is loaded, etc. A modder can subscribe event handlers to each of these events, and then perform their desired behavior inside these event handlers.
 
-Example: An event can be used to perform an action when a Vanilla stick is right-clicked.
+Events are fired on their respective event bus. The most important bus is `NeoForge.EVENT_BUS`. Besides that, during startup, a mod bus is spawned for each loaded mod and passed into the mod's constructor; mod bus event handlers can run in parallel, dramatically increasing startup speed. See [below][modbus] for more information.
```

Comment:
```suggestion
Events are fired on their respective event bus. The most important bus is `NeoForge.EVENT_BUS`. Besides that, during startup, a mod bus is spawned for each loaded mod and passed into the mod's constructor. Some mod bus event handlers can run in parallel, dramatically increasing startup speed. See [below][modbus] for more information.
```

Otherwise, it sounds like all mod bus methods run in parallel, which I don't believe is the case.

Reply:
Hence the "can run in parallel" (as opposed to "do run in parallel"), but will adjust to better wording.

Reply:
I understood the sentiment; however, my brain interpreted it as all regardless.